test/OpenIddict.Abstractions.Tests/Primitives/OpenIddictExtensionsTests.cs

```diff
@@ -3010,6 +3009,60 @@ public void ClaimsPrincipal_SetClaimWithJsonElement_RemovesEmptyClaim()
         Assert.Null(principal.GetClaim("type"));
     }
 
+    [Fact]
+    public void ClaimsIdentity_SetClaimWithJsonElement_Undefined()
+    {
+        // Arrange
+        var identity = new ClaimsIdentity();
+
+        // Act
+        identity.SetClaims("type", default(JsonElement));
```

Comment:
Nit: the method used here doesn't match the name of the test: `SetClaims` vs `SetClaim`.
AFAICT, the other test methods are affected too.

Reply:
Make sense